### ISSUE TITLE
Does not work in Firefox

### ISSUE THREAD

**Issue by pbrisbin** [thoughtbot/carnival#248]
Steps:

- Firefox 37.0.1-2
- https://carnivalapp.io
- Hover, click "Leave comment"
- See in console:

```
InvalidAccessError: A parameter or an operation is not supported by the underlying object
```

Based this: http://stackoverflow.com/questions/16677893/evil-firefox-error-a-parameter-or-an-operation-is-not-supported-by-the-under, it's because of the `async: false` on the call for user information (see #170). One solution may be here: http://stackoverflow.com/questions/16668386/cors-synchronous-requests-not-working-in-firefox

**Comment by pbrisbin** [comment]
Reopening, the fix was not a fix and breaks other browsers. We need to instead fix #170 to get things working in Firefox.

**Comment by jferris** [comment]
This seems to be working in Firefox now. I think this was fixed by #170.
